### PR TITLE
Various grammar fixes to effects.json

### PR DESCRIPTION
**Pull request by ToxiClay**
#### Summary

SUMMARY: None

#### Purpose of change

This change introduces several grammar changes to `effects.json`.

#### Describe the solution

In the course of filing PR #46245, I found errors in this file that needed to be changed.

#### Describe alternatives you've considered

No alternatives.

#### Testing

JSON linting performed; syntax error encountered. Unknown impact.

### CODE REVIEW

File: data/json/effects.json [hunk 1]
```diff
@@ -535,7 +535,8 @@
     "type": "effect_type",
     "id": "foodpoison",
     "name": [ "Food Poisoning" ],
-    "desc": [ "Your stomach is extremely upset, and you are quite nauseous." ],
+    "desc": [ "Your stomach is extremely upset, and you are quite nauseated." ],
+    "//": "In formal usage, 'nauseous' technically means 'to cause nausea [in others],' not 'feeling sick.'"
```
Comment (ZhilkinSerg) [inline data/json/effects.json:539]:
```suggestion
    "//": "In formal usage, 'nauseous' technically means 'to cause nausea [in others],' not 'feeling sick.'",
```

File: data/json/effects.json [hunk 2]
```diff
@@ -1983,14 +1984,14 @@
     "name": [ "Mild hypovolemic shock", "Moderate hypovolemic shock", "Advanced hypovolemic shock", "Severe hypovolemic shock" ],
     "desc": [
       "You've lost some blood and look somewhat pale.",
-      "You've lost large amount of blood, and you're not feeling well.",
+      "You've lost a large amount of blood, and you're not feeling well.",
       "You've lost an awful lot of blood, and your condition is severe.  Seek medical attention.",
       "You've lost tremendous amount of blood, and you're standing on death's door.  Transfusion might save you."
     ],
     "apply_message": "You have lost lot of blood, and your condition worsens.",
```
Comment (actual-nh) [inline data/json/effects.json:1991]:
```suggestion
    "apply_message": "You have lost a lot of blood, and your condition worsens.",
```

File: data/json/effects.json [hunk 3]
```diff
@@ -1983,14 +1984,14 @@
...
     "remove_message": "You are no longer in shock.",
     "decay_messages": [
-      [ "Your blood level increase, and you feel better, but still look pale.", "good" ],
+      [ "Your blood levels increase, and you feel better; but still, you look pale.", "good" ],
```
Comment (actual-nh) [inline data/json/effects.json:1994]:
```suggestion
      [ "Your blood volume increases, and you feel better; but still, you look pale.", "good" ],
```
"Levels" is what I would use to describe how much of something is in the blood - blood calcium levels, blood lead levels...

File: data/json/effects.json [hunk 4]
```diff
@@ -1983,14 +1984,14 @@
...
       [ "Your feel stronger as your blood levels starts to improve.", "good" ],
```
Comment (actual-nh) [inline data/json/effects.json:1995]:
```suggestion
      [ "You feel stronger as your blood volume starts to improve.", "good" ],
```

**Review by actual-nh — COMMENTED** [review]
Good!

File: data/json/effects.json (hunk 3 above)
Comment (ToxiClay) [reply data/json/effects.json:1994]: Yeah, you're right. I didn't find it particularly wrong on a first pass, but now I agree with you.

**Pull request merged**